packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue

```diff
@@ -599,7 +631,7 @@ onUnmounted(() => {
 .button-column {
 	display: flex;
 	flex-direction: column;
-	padding: 1rem 0rem 0.5rem 0rem;
+	padding: 0.5rem 0rem 0.5rem 0rem;
```

Comment:
Use css variables like `var(--gap-small)` found in `variables.scss`. Also this can be shortened to `0.5rem 0` or `var(--gap-small) 0`